docs/source/federated-types/federated-directives.mdx

```diff
@@ -346,6 +362,25 @@ For more information, see [Migrating entities and fields](../entities-advanced/#
 </td>
 </tr>
 
+<tr class="required">
+<td>
+
+##### `label`
+
+`String`
+</td>
+<td>
+
+> ⚠️ **This argument is available in Apollo Federation 2.7 and later.**
+> **It is an [Enterprise feature](/router/enterprise-features) of the Apollo Router** and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).<br/>
+>
+> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+
+**Optional.** The percentage of traffic for this field (randomly chosen from a uniform distribution) that is resolved by this subgraph. The remaining percentage is resolved by the other ([from](#from)) subgraph.
```

Comment:
`label` doesn't have to be a `percent(x)` value, it can also be an arbitrary string. We handle the `percent(x)` case specifically, but users will be able to handle resolving arbitrary labels to their respective T/F values in rhai script/coprocessor.

I don't think we need to capture all of that info here, but we shouldn't say that a label specifically matches to a percentage.